src/main/java/frc/robot/RobotContainer.java

```diff
@@ -24,6 +27,15 @@ public RobotContainer() {
     private void configureBindings() {
         drive.registerTelemetry(logger::telemeterize);
         drive.setDefaultCommand(new DriveWithJoysticks(drive, leftJoystick, rightJoystick));
+
+        // SYS ID
+        /* Bindings for drivetrain characterization */
+        /* These bindings require multiple buttons pushed to swap between quastatic and dynamic */
+        /* Back/Start select dynamic/quasistatic, Y/X select forward/reverse direction */
+        masher.back().and(masher.y()).whileTrue(drive.sysIdDynamic(Direction.kForward));
+        masher.back().and(masher.x()).whileTrue(drive.sysIdDynamic(Direction.kReverse));
+        masher.start().and(masher.y()).whileTrue(drive.sysIdQuasistatic(Direction.kForward));
+        masher.start().and(masher.x()).whileTrue(drive.sysIdQuasistatic(Direction.kReverse));
```

Comment:
Can we configure these in test mode only

Reply:
yeah, i can add that